### PR TITLE
[Web - MPC] Add Near and Solana docs

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

File: docs/connect-blockchain/near/near.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+blockchain data, and managing user authentication within Solana-based applications.
+
+With Web3Auth's native support for Near and the `ed25519` curve, developers can streamline their
+development process and harness the full potential of blockchain technology within their Near-based
+projects.
+
+This documentation provides a straightforward guide for developers looking to implement Near
+connection quickly and effortlessly across various platforms.
+
+export const Solana = [
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:26]: This should Near

File: docs/connect-blockchain/near/near.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+description: "Integrate Web3Auth with the Near Blockchain | Documentation - Web3Auth"
+---
+
+import Tiles from "@theme/Tiles";
+
+For Near integration, Web3Auth offers an `ed25519` private key specifically tailored for the Solana
+blockchain. This cryptographic key conforms to the standards required for secure authentication and
+transaction signing on Solana's network.
+
+Integrating Web3Auth with Solana is straightforward and efficient. Developers can leverage
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:14]: Near not Solana

File: docs/connect-blockchain/near/near.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+title: Integrate Web3Auth with the Near Blockchain
+image: "banners/near.png"
+hide_table_of_contents: true
+description: "Integrate Web3Auth with the Near Blockchain | Documentation - Web3Auth"
+---
+
+import Tiles from "@theme/Tiles";
+
+For Near integration, Web3Auth offers an `ed25519` private key specifically tailored for the Solana
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:10]: Near

File: docs/connect-blockchain/near/near.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+blockchain. This cryptographic key conforms to the standards required for secure authentication and
+transaction signing on Solana's network.
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:12]: Near

File: docs/connect-blockchain/near/near.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Integrating Web3Auth with Solana is straightforward and efficient. Developers can leverage
+Web3Auth's functionalities to establish secure connections with Solana nodes, enabling seamless
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:15]: Call them Ed25519/ EDDSA nodes rather

File: docs/connect-blockchain/near/near.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+interaction with the blockchain. This includes tasks such as submitting transactions, querying
+blockchain data, and managing user authentication within Solana-based applications.
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:17]: Near

File: docs/connect-blockchain/near/near.mdx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        key: "web-mpc",
+        title: "Web - MPC",
+        icon: "logo-js.png",
+        path: "/connect-blockchain/near/web-mpc",
+      },
+    ],
+  },
+];
+
+<Tiles tileGroups={Solana} />
```
Comment (yashovardhan) [inline docs/connect-blockchain/near/near.mdx:47]: Near. Basically don't have much mention of Solana in this page.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
Some of my recent changes might conflict a few things as well. Please make sure about that. Overall it is fine, just fix the near page.

File: docs/connect-blockchain/near/near.mdx (hunk 5 above)
Comment (AyushBherwani1998) [reply docs/connect-blockchain/near/near.mdx]: Have kept nodes, because ed25519 nodes would be confusing, considering not everyone knows about curves.

**Comment by yashovardhan** [comment]
Please resolve the conflicts

**Review by shahbaz17 — APPROVED** [review]
lgtm

**Review by shahbaz17 — APPROVED** [review]
lgtm

**Pull request merged**